test/metabase/query_processor_test/offset_test.clj

```diff
@@ -43,106 +43,6 @@
                 [->local-date 2.0 2.0]
                 (qp/process-query query))))))))
 
-(deftest ^:parallel offset-expression-test
```

Comment:
These tests were failing after re-enabing nesting for expressions in order-by.

Per slack thread [here](https://metaboat.slack.com/archives/C0645JP1W81/p1742409338657409?thread_ts=1742317450.874479&cid=C0645JP1W81), we no longer allow offset outside of aggregations, so removed these tests.